test/ventilation-test.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * ventilation-test.c
+ *
+ * test the function contain in ventilation.c
+ */
+
+#include "application/ventilation.h"
+#include "stubs/systemInfo_stub.h"
+#include "stubs/valve_stub.h"
+#include "testUtil.h"
+
+
+void reset_to_inspiration_start();
+
+void startVentilation_openInspiratoryValve() {
+  int* times = (int*) malloc(2*sizeof(int)); 
```

Comment:
I don't like the use of malloc in this test. Can you find a different solution? See my suggestions above.